licensedb/filer/filer.go

```diff
@@ -108,7 +108,7 @@ type gitFiler struct {
 
 // FromGitURL returns a Filer that allows accessing all the files in a Git repository given its URL.
 func FromGitURL(url string) (Filer, error) {
-	repo, err := git.Clone(memory.NewStorage(), nil, &git.CloneOptions{URL: url})
+	repo, err := git.Clone(memory.NewStorage(), nil, &git.CloneOptions{URL: url, Depth: 1})
```

Comment:
As this seems to always use a reference to a default remote branch - there could not be a situation when the shallow clone does not include the branch later on.

Otherwise it would require `--no-single-branch` equivalent.

It would be nice to mention shallow behaviour in the go doc string.